app/build.gradle

```diff
@@ -253,6 +253,9 @@ dependencies {
     implementation "com.google.firebase:firebase-analytics:$versions.firebase_analytics"
     implementation "com.google.firebase:firebase-crashlytics:$versions.firebase_crashlytics"
 //    androidTestImplementation "androidx.navigation:navigation-testing:2.3.0-alpha05" //TODO change to stable when available
+
```

Comment:
Afaik this lib is only required if you have string timestamps with timezones, which we don't have, we always have utc timezones from the backend.

Reply:
It is needed to load the timezones database. And we need timezone data to calculate local times from UTC.

Reply:
Afaik this is only if the timestamp contains a timezone else you can use the timezone of the device (and for that the lib is not required). I am fine if you want to use the library, but everything that we want to do right now (e.g. display a utc timestamp in the device local time) can be done without.

Also the lib has a warning that the development is winding down and that a new feature should be used. Was this evaluated?

Reply:
Desugaring seems to work and is implemented in b837a17ae7d09b36f417b87be5ffb5cadb581ab9